_authors/auclair.md

```diff
@@ -2,3 +2,6 @@
 title: "Pascal Auclair"
 ---
 
+Pascal Auclair has been immersed in Buddhist practice and study since 1997, sitting retreats in Asia and America with revered monastics and lay teachers. He has been mentored by Joseph Goldstein and Jack Kornfield at the Insight Meditation Society (IMS) in Massachusetts and Spirit Rock Meditation Center in California, where he is now enjoying teaching retreats. Pascal teaches in North America and in Europe. He is a co-founder of True North Insight and one of TNI’s Guiding Teachers.
```

Comment:
I think can remove ", sitting retreats in Asia and America with revered monastics and lay teachers" as this is implied by "immersed"